src/treemodel.cpp

```diff
@@ -91,6 +91,26 @@ QVariant TreeModel::data(const QModelIndex &index, int role) const
             return item->Data(col);
             break;
         }
+        case Qt::ToolTipRole:
+        {
+            if (col == COL::Value)
+            {
+                const int ToolTipLimit = 500;
+                QString tip = GetValueFullString(index);
+                if (tip.size() > ToolTipLimit)
+                {
+                    tip.truncate(ToolTipLimit);
+                    tip += "...";
```

Comment:
I I often find myself in need of looking at few characters at the end (like "rows"). Perhaps we can take first 450 and last 50 or something long these lines?